test/js-api/jsapi.js

```diff
@@ -683,48 +710,75 @@ test(() => {
     const instantiateDesc = Object.getOwnPropertyDescriptor(WebAssembly, 'instantiate');
     const instantiate = WebAssembly.instantiate;
     assert_equals(instantiate, instantiateDesc.value);
-    assert_equals(instantiate.length, 2);
+    assert_equals(instantiate.length, 1);
```

Comment:
Note that Ecma 262 Sec. 17 continues saying "Optional parameters (which are indicated with brackets:  [ ]) or rest parameters (which are shown using the form «...name») are not included in the default argument count.". So also the following Note 2 for additional clarification. So we should only count `bytes`.